tools/refguide_check.py

```diff
@@ -119,6 +119,8 @@
     'doc/release',
     'doc/source/release',
     'c-info.ufunc-tutorial.rst',
+    'c-info.python-as-glue.rst',
+    'getting-started.rst',
```

Comment:
I got scared for a second, until I realized that `gettting-started.rst` is not the general beginner guide, rather the one inside the `f2py` directory. Could you rename the file to `f2py.getting-started.rst` or something that would prevent a future conflict with a `getting-started.rst` file for another task?

Reply:
Sure. Here, as a contributor I hesitated to rename a file.